test/build_samples.go

```diff
@@ -80,3 +80,80 @@ spec:
     image: image-registry.openshift-image-registry.svc:5000/example/buildpacks-app
   timeout: 30s
 `
+
```

Comment:
Perhaps a note or Github issue for the future consideration - use go-bindata to manage test data.

https://github.com/go-bindata/go-bindata